### ISSUE TITLE
Can't receive order confirmation email after unsubscribe from newsletter

### ISSUE THREAD

**Issue by AlexanderRakov** [sailthru/sailthru-magento2-extension#25]
**Preconditions:**
Email address is registered in SailThru (example, subscribed to newsletter)
Got emails from site
**Steps to reproduce:**
Unsubscribe from newsletter using Opt Back In button
After that create Magento order on same email address
**Expected result:**
Receive order confirmation email
**Actual result:**
No Confirmation email on email box
In SailThru Admin this email is marked as opting out
Following error is appeared in log:
```
(
  'error' => 33,
  'errormsg' => 'Email alexander.rakov@cyberhull.com has opted out of delivery from Magento Generic',
)
```

In other words press Opt Back In button disable all emails for the email address at all
![current catalog dev - opt out - google chrome](https://user-images.githubusercontent.com/7290540/27688394-24121af4-5ce3-11e7-93f9-9c5f510c1107.jpg)

**Comment by sshymko** [comment]
Any update on this?
Opted-out customers cannot receive transactional emails, including order confirmation.

**Comment by silverman63** [comment]
Hi all, apologize for delay. @sshymko I believe @nickgundry has started offline conversation with you about this. We are planning to address this by separating all templates. Once templates are separate, they will be able to be marked as basic and be sent to opt'd out users.

**Comment by silverman63** [comment]
Alternatively, for order confirmation, you can currently choose to use a Sailthru template. If you use a Sailthru template, you can have that submitted to be basic.

**Comment by silverman63** [comment]
Closing, as this should be resolved. @sshymko please let me know if anything here is outstanding.
